services/export/shared/csvExport.js

```diff
@@ -303,6 +303,9 @@ const downloadTransactionHistory = async ({ filename }) => {
 	const isFileExists = await staticFiles.exists(filename);
```

Comment:
Please add a regex test (`CSV_EXPORT_FILENAME.test(filename)`) at the beginning to validate the `filename`.